iomgr/lib/dbengine/parser/AntlrHelpers.h

```diff
@@ -14,6 +14,24 @@ constexpr std::size_t kInvalidNodeType = std::numeric_limits<std::size_t>::max()
 
 namespace helpers {
 
+/**
+ * Convert double quote to single quote from string.
+ * NOTE: in SQL, a single quote is escaped with a single quote.
+ * @param s A string.
+ * @return Same string.
+ */
+inline std::string& fixSingleQuotes(std::string& s)
```

Comment:
I just noticed this one sits in the header file. This is too complicated function to be inlined. So "inline" must be removed, header must contain only documentation comment and function declaration, function definition must be moved into the corresponding cpp file (AntlrHelpers.cpp).